### DISCUSSION TITLE
MACS3 - custom genome

### DISCUSSION THREAD

**Discussion by PycnopodiaD** [macs3-project/MACS#529] [General]
Hi, I would like to use macs3 with a custom genome, or a genome that is not found on the list of the 4 genomes that can be used, is that possible? I tried using:

-g /path/to/the/genome/file.fa --2.5e9

But I just got an error message

macs3: error: unrecognized arguments: --2.5e9

If I leave that out I get;

ERROR:root:Available shortcuts of effective genome sizes are hs,mm,ce,dm

Maybe the syntax is just wrong? I don't know but it would be nice to be able to use another genome because I am not working on those 4 genomes.

**Comment by mrendleman** [comment]
The -g parameter just needs a number, not the path to your genome fasta. If your effective genome size is 2.5e9, use: `-g 2.5e9`

If you haven't calculated the effective genome size for your custom assembly, the [documentation of deepTools](https://deeptools.readthedocs.io/en/develop/content/feature/effectiveGenomeSize.html) has a nice overview and links to several tools you can use to calculate it.
